umm-spec-lib/resources/json-schemas/collection/umm/v1.10/umm-cmn-json-schema.json

```diff
@@ -863,10 +863,7 @@
         },
         "DataType": {
           "description": "The datatype of the Characteristic/attribute.",
-          "type": "string",
-          "minLength": 1,
-          "maxLength": 80,
-          "pattern": "[\\w]{1,79}"
+          "$ref": "#/definitions/DataTypeEnum" 
```

Comment:
We need to keep the latest version of the umm-cmn-json-schema.json in sync for all concept types. When the umm-cmn-json-schema.json is updated for one concept type, the corresponding files should be updated for all concept types.

Reply:
That's bad. now we have same code kept in multiple places?

Reply:
Yes, the common schema is shared and we should keep it consistent, and I couldn't think of a better way of doing this.

Reply:
Done.